network_generation.py

```diff
@@ -1,13 +1,34 @@
 import pandapower as pp
+import numpy as np
 
 
 def get_net(config):
     if config.env_name == 'Six_Bus_POC':
         return six_bus(config.vn_high, config.vn_low, config.length_km,
-                       config.std_type)
+                       config.std_type, config.battery_locations, config.init_soc,
+                       config.energy_capacity, config.static_feeds)
 
 
-def six_bus(vn_high=20, vn_low=0.4, length_km=0.03, std_type='NAYY 4x50 SE'):
+def add_battery(net, bus_number, p_init, energy_capacity, init_soc=0.5,
+                max_p=50, min_p=-50, eff=1.0, capital_cost=0, min_e=0.):
+    """Change the network by adding a battery / storage unit. """
+    pp.create_storage(net, bus_number, p_init, energy_capacity,
+                      soc_percent=init_soc, max_p_kw=max_p, min_p_kw=min_p,
+                      min_e_kwh=min_e)
+    # if 'eff' not in net.storage.columns:
+    #     net.storage['eff'] = eff
+    # else:
+    idx = net.storage.index[-1]
```

Comment:
Were you able to test that this worked correctly?

Reply:
Yes it tests okay, I'll remove the commented out older parts